subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf52840.h

```diff
@@ -404,16 +404,25 @@ static inline u32_t hal_radio_phy_mode_get(u8_t phy, u8_t flags)
 	case BIT(0):
 	default:
 		mode = RADIO_MODE_MODE_Ble_1Mbit;
+
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
```

Comment:
@cvinayak are these workaround not applicable when we set PHY to 1M or 2M?

Reply:
this statements are to revert workaround under asymmetric PHY (1M/2M and Coded), hence not needed if Coded not supported.